servant.core

```diff
@@ -8,6 +8,10 @@ filesets:
     file_type : verilogSource
     depend : ["fusesoc:utils:generators"]
 
+  service_no_pll:
+    files: [servant/service_no_pll.v]
+    file_type : verilogSource
+
```

Comment:
Not sure if you would prefer this to be `service_go_board` or this more generic `service_no_pll`. With a board-specific `service_go_board`, I could also pull the other 3 LEDs low, like I did with `blinky`, instead of having them float without pull-ups.

Reply:
Yeah, combine the new toplevel and the PCF into a board-specific toplevel. If you do that it makes sense to pull down the other LEDs too. Does the Go board have an UART output as well? If so, you could connect the GPIO output from servant tp both so that the single GPIO can ve used both for blinking LEDs and bitbanging a UART (alsy handy if you want to easily see activity on the UART)

Reply:
Set `servant.q` to both LED1 *and* UART TX? I can add that.